src/components/explat/experiments/single-view/results/ExperimentResults.tsx

```diff
@@ -394,10 +394,14 @@ export default function ExperimentResults({
     experiment,
     primaryMetricLatestAnalysesByStrategy,
   )
-  const experimentHealthIndicators = [
-    ...Analyses.getExperimentParticipantHealthIndicators(experiment, experimentParticipantStats),
-    ...Analyses.getExperimentHealthIndicators(experiment),
-  ]
+
+  const experimentHealthIndicators =
+    experiment.status === Status.Disabled && !experiment.startDatetime
```

Comment:
```suggestion
    experiments.shouldHaveResults(experiment)
```
This needs an explanation, I think the best way is to add a function, an alternative is to add a comment.

If we use the function we should explain that missing a start datetime means the experiment went straight from staging to disabled.